src/components/OSMToggleButton/index.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div id='displayOSM'>
+        <v-btn id="osm"
+            absolute
+            bottom
+            right
+            fab
+            small
+            @click="displayOSM = !displayOSM"
+        >
```

Comment:
```suggestion
            @click="displayOSM = !displayOSM"
            @dblclick.stop=";"
        >
```
でダブルクリックの反応止められると思います。

Reply:
できました。ありがとうございます。